pkg/volume/nfs/nfs.go

```diff
@@ -241,6 +242,51 @@ func (nfsMounter *nfsMounter) SetUp(mounterArgs volume.MounterArgs) error {
 	return nfsMounter.SetUpAt(nfsMounter.GetPath(), mounterArgs)
 }
 
+// Generate commands for configuring the UID and GID for NFS access.
+func setupCommand(name string) string {
+	pathname := ` -Path "HKLM:\SOFTWARE\Microsoft\ClientForNFS\CurrentVersion\Default\" -Name ` + name
+	getcmd := "Get-ItemProperty" + pathname
+	newcmd := "New-ItemProperty" + pathname + "-Value 0 -PropertyType DWORD"
+	bothcmd := getcmd + "; if ( !$? ) { " + newcmd + " }"
+	return bothcmd
+}
+
+// Install NFS client and configure the UID and GID for NFS access.
+func setupNFSonWindows() error {
+	cmd := exec.Command("powershell", "/c", `Install-WindowsFeature -Name NFS-Client`)
+	output, err := cmd.CombinedOutput()
+	klog.Infof("NFS windows setup %v %v", string(output), err)
+	if err != nil {
+		return err
+	}
+	command := setupCommand("AnonymousUid")
+	output, err = exec.Command("powershell", "/c", command).CombinedOutput()
+	klog.Infof("NFS windows setup %v %v", string(output), err)
+	if err != nil {
+		return err
+	}
+	command = setupCommand("AnonymousGid")
+	output, err = exec.Command("powershell", "/c", command).CombinedOutput()
+	klog.Infof("NFS windows setup %v %v", string(output), err)
+	if err != nil {
+		return err
+	}
+
+	cmd = exec.Command("powershell", "/c", `nfsadmin client stop`)
```

Comment:
`nfsadmin client stop` takes a few seconds, so if `AnonymousGid` is already configured, no need to restart `nfsadmin client`